iree_tests/README.md

```diff
@@ -1,5 +1,39 @@
 # IREE Tests
 
+## Quickstart instructions
```

Comment:
This duplicates the setup at https://github.com/nod-ai/SHARK-TestSuite/tree/main/iree_tests#common-venv-setup-with-deps . Want to merge the two?

* Keep it as "quickstart" near the top
* Keep as a single code block for easier copy/paste
* Add a line or two for local build dir and `$PATH` vs python packages
* Move the overview text ("A [conftest.py](https://github.com/nod-ai/SHARK-TestSuite/blob/main/iree_tests/conftest.py) file collects test cases from subdirectories, wrapping ...") to the newly labeled "About" section